src/Widgets/DataSearchCountWidget/DataSearchCountWidgetEditingConfig.ts

```diff
@@ -0,0 +1,51 @@
+import { provideEditingConfig } from 'scrivito'
+import { DataSearchCountWidget } from './DataSearchCountWidgetClass'
+import thumbnail from './thumbnail.svg'
+
+provideEditingConfig(DataSearchCountWidget, {
+  title: 'Data Search Count',
+  thumbnail,
+  attributes: {
+    loadingHeadline: { title: 'Headline while loading results' },
+    headline0: { title: 'Headline for 0 results' },
+    headline1: { title: 'Headline for 1 result' },
+    headline: {
+      title: 'Headline for multiple results',
+      description:
+        'The placeholder __count__ represents the total number of results.',
+    },
+  },
+  properties: ['loadingHeadline', 'headline0', 'headline1', 'headline'],
+  initialContent: {
+    headline: '__count__ search results',
+    headline0: 'No search results',
+    headline1: '1 search result',
+    loadingHeadline: 'Search results',
+  },
+  validations: [
+    [
+      'headline',
+      (headline) =>
+        !headline && {
+          severity: 'warning',
+          message: 'The headline should be set.',
+        },
+    ],
+    [
+      'headline0',
+      (headline) =>
+        !headline && {
+          severity: 'warning',
+          message: 'The headline for 0 results should be set.',
+        },
+    ],
+    [
+      'headline1',
+      (headline) =>
+        !headline && {
+          severity: 'warning',
+          message: 'The headline for 1 result should be set.',
+        },
+    ],
+  ],
```

Comment:
* [ ] FYI as the component is happy and fully functioning without h0 and h1 we could drop these warnings.

Reply:
I think I remember how added the code originally 🙄: https://github.com/Scrivito/scrivito-portal-app/pull/357/commits/1ec43ee921146cc4d238bb949dd48c5826a71006#diff-bc72c026851dc07f2844efe663f800f856d2c52f02407caf602d951ad88fa353R42-R74

I think the validation is useful. Yes, you can leave the attribute empty, but then nothing is shown at all.

Reply:
> then nothing is shown at all.

True. I misread `const attribute = attributes[totalCount] || 'headline'` but when I (mis)read it I thought falling back to *the content of* `headline` makes sense.